include/seqan/translation/translation.h

```diff
@@ -312,8 +312,8 @@ _translateImplLoopOMPWrapper(TTarget & target,
 {
     SEQAN_OMP_PRAGMA(parallel for schedule(dynamic))
     for (int64_t i = 0; i < static_cast<int64_t>(length(target)); ++i)
-        _translateImplLoop(target, i, source, GeneticCode<CODE_SPEC>(),
-                           Frames_<frames>());
+        if (length(source[i/frames]) >= 3)
```

Comment:
Please add braces around the for-loop body.

Reply:
it's all one line statements, that doesn't violate our rules :point_up:

Reply:
😒